.changeset/khaki-tigers-battle.md

```diff
@@ -0,0 +1,7 @@
+---
+"@salt-ds/core": minor
+---
+
+**_Significant styling change_**
```

Comment:
Is this ok for the changeset? The styling is no longer available, but we don't consider that breaking.. @joshwooding